### PR TITLE
fix: thorchain LP sym withdraw RUNE support detection

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Fixes the case where there *is* a RUNE position address for a withdraw (i.e, it's a sym position), but no RUNE account in the portfolio (i.e, withdrawing asymmetrically on the asset side, while having accounts for the asset connected)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/7634

## Risk

> High Risk PRs Require 2 approvals

Low to none

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

Ensure asym asset withdraws of a sym position correctly guard against RUNE chain not supported

### Engineering

- ^

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

- ^

## Screenshots (if applicable)

Screenshot 2024-08-27 at 18 52 21
Screenshot 2024-08-27 at 18 52 08

### CODE REVIEW

File: src/lib/utils/thorchain/hooks/useSendThorTx.tsx [hunk 1]
```diff
@@ -270,10 +270,8 @@ export const useSendThorTx = ({
     if (!accountId) return
     if (!transactionType) return
     if (!estimateFeesArgs) return
-    if (isToken(asset.assetId) && !inboundAddressData) return
-
-    await checkLedgerAppOpenIfLedgerConnected(asset.chainId)
     if (accountNumber === undefined) return
+    if (isToken(asset.assetId) && !inboundAddressData) return
```
Comment (gomesalexandre) [inline src/lib/utils/thorchain/hooks/useSendThorTx.tsx:274]: diff is drunk, `accountNumber` has been brought back up, and `checkLedgerAppOpenIfLedgerConnected()` back down.

Comment (kaladinlight) [reply src/lib/utils/thorchain/hooks/useSendThorTx.tsx:274]: seeing this after commenting on the previous pr. that answers that.

**Review by kaladinlight — APPROVED** [review]
Unrelated to ledger ack really, but working as described

File: src/lib/utils/thorchain/hooks/useSendThorTx.tsx (hunk 1 above)
Comment (gomesalexandre) [reply src/lib/utils/thorchain/hooks/useSendThorTx.tsx:274]: And diff looks good now 🎉

**Pull request merged**